_includes/header.html

```diff
@@ -3,29 +3,13 @@
     <nav id="page-links">
       <img tabindex="0" id="hamburger-button" alt="Menu Button" src="/uploads/hamburger.svg" class="mobile-only">
       <div class="desktop-only">
-        {% if page.url == "/" %}
-          <a href="/" class="text-weight-bold">Home</a>
-        {% else %}
-          <a href="/">Home</a>
-        {% endif %}
-
-        {% if page.url == "/about/" %}
-          <a href="/about" class="text-weight-bold">About</a>
-        {% else %}
-          <a href="/about">About</a>
-        {% endif %}
-        
-        {% if page.url == "/now/" %}
-          <a href="/now" class="text-weight-bold">Now</a>
-        {% else %}
-          <a href="/now">Now</a>
-        {% endif %}         
-        
-        {% if page.url == "/contact/" %}
-          <a href="/contact" class="text-weight-bold">Contact</a>
-        {% else %}
-          <a href="/contact">Contact</a>
-        {% endif %}
+        {% for item in site.navigation_links %}
+          {% if page.url == item.url %}
+            <a href="{{ item.url }}" class="text-weight-bold">{{ item.title }}</a>

```

Comment:
Maybe we should make the class more semantic like `selected` in case the styling changes down the road?

```suggestion
            <a href="{{ item.url }}" class="selected">{{ item.title }}</a>
```

Reply:
As you like, I don't have a strong opinion on this.